tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -25,107 +22,264 @@ Create VCH - custom base disk
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - URL without user and password
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
```

Comment:
understand it does not impact performance, but refactor a little bit to call it in case tear down, instead of in every case, might be better.